src/Preview/Preview.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { node, string, oneOf } from 'prop-types';
+
+import { dataHooks } from './constants';
+import style from './Preview.st.css';
+
+/** Preview content widget*/
+class Preview extends React.PureComponent {
+  static displayName = 'Preview';
+
+  static propTypes = {
+    /** Preview data hook*/
+    dataHook: string,
+
+    /** Preview type */
+    type: oneOf(['blank', 'browserBar', 'mobile']),
+
+    /** Preview background color*/
+    backgroundColor: oneOf(['grey', 'gradient', 'custom']),
+
+    /** Content area contour*/
+    borderType: oneOf(['shadow', 'solid']),
+
+    /** Node to preview */
+    children: node.isRequired,
+  };
+
+  static defaultProps = {
+    type: 'blank',
+    backgroundColor: 'grey',
+    borderType: 'shadow',
+  };
+
+  render() {
+    const {
+      dataHook,
+      type,
+      backgroundColor,
+      borderType,
+      children,
+    } = this.props;
+
+    return (
+      <div {...style('root', { backgroundColor })} data-hook={dataHook}>
```

Comment:
```suggestion
      <div {...style('root', { backgroundColor }, this.props)} data-hook={dataHook}>
```
passing the 3rd arg lets consumers override the style.